### ISSUE TITLE
PackError: Could not pack item 7: ('I', 100000000000) - Market Community

### ISSUE THREAD

**Issue by xoriole** [Tribler/tribler#4147]
Multiple reports obtained of PackError in market community.
```python
Traceback (most recent call last):
File "/usr/share/tribler/TriblerGUI/event_request_manager.py", line 133, in on_read_data
raise RuntimeError(json_dict["event"]["text"])
RuntimeError: [Failure instance: Traceback: <class 'Tribler.pyipv8.ipv8.messaging.serialization.PackError'>: Could not pack item 7: ('I', 100000000000)
error: 'I' format requires 0 <= number <= 4294967295
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:653:_runCallbacks
/usr/share/tribler/Tribler/community/market/community.py:220:on_peers
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:459:callback
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:567:_startRunCallbacks
--- <exception caught here> ---
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:653:_runCallbacks
/usr/share/tribler/Tribler/community/market/community.py:836:on_peer_address
/usr/share/tribler/Tribler/pyipv8/ipv8/lazy_community.py:171:_ez_pack
/usr/share/tribler/Tribler/pyipv8/ipv8/messaging/serialization.py:270:pack_multiple
/usr/share/tribler/Tribler/pyipv8/ipv8/messaging/serialization.py:264:pack_multiple
/usr/share/tribler/Tribler/pyipv8/ipv8/messaging/serialization.py:248:pack
/usr/share/tribler/Tribler/pyipv8/ipv8/messaging/serialization.py:172:pack
]
```
```python
Traceback (most recent call last):
File "/usr/share/tribler/TriblerGUI/event_request_manager.py", line 133, in on_read_data
raise RuntimeError(json_dict["event"]["text"])
RuntimeError: [Failure instance: Traceback: <class 'Tribler.pyipv8.ipv8.messaging.serialization.PackError'>: Could not pack item 7: ('I', 100000000000)
error: 'I' format requires 0 <= number <= 4294967295
/usr/lib/python2.7/dist-packages/twisted/internet/base.py:1252:mainLoop
/usr/lib/python2.7/dist-packages/twisted/internet/base.py:851:runUntilCurrent
/usr/share/tribler/Tribler/community/market/community.py:841:get_address
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:310:addCallbacks
--- <exception caught here> ---
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:653:_runCallbacks
/usr/share/tribler/Tribler/community/market/community.py:836:on_peer_address
/usr/share/tribler/Tribler/pyipv8/ipv8/lazy_community.py:171:_ez_pack
/usr/share/tribler/Tribler/pyipv8/ipv8/messaging/serialization.py:270:pack_multiple
/usr/share/tribler/Tribler/pyipv8/ipv8/messaging/serialization.py:264:pack_multiple
/usr/share/tribler/Tribler/pyipv8/ipv8/messaging/serialization.py:248:pack
/usr/share/tribler/Tribler/pyipv8/ipv8/messaging/serialization.py:172:pack
]
```

**Comment by qstokkink** [comment]
https://github.com/Tribler/tribler/blob/devel/Tribler/community/market/community.py#L835

Seems like the timeout value.

**Comment by synctext** [comment]
Note for prosperity: this bug resides in the sender. Difficult to prevent type error. A too large number is attempted to be converted into a 32-bit unsigned int. Results in Traceback. IPv8 is not yet called, so can't help.

**Comment by devos50** [comment]
Fixed in the Tribler 7.1.4 release which can be found [here](https://github.com/Tribler/tribler/releases/tag/v7.1.4).
